src/lib/sw_engine/tvgSwShape.cpp

```diff
@@ -465,6 +465,7 @@ void shapeDelOutline(SwShape* shape, uint32_t tid)
 void shapeReset(SwShape* shape)
 {
     rleReset(shape->rle);
+    rleReset(shape->strokeRle);
```

Comment:
you can raise a patch for this change with another one.

Reply:
done. Please check #228